keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -364,7 +364,7 @@ metadata:
 
 1. `name`: the BA name is generated by COSI in this format: _"<BAR.namespace*>-<BAR.name*>-<uuid>"_. The uuid is unique within a cluster. `*` the first 10 characters of the namespace and the name are used.
 1. `labels`: added COSI.  Key’s value should be the provisioner name. Characters that do not adhere to [Kubernetes label conventions](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) will be converted to ‘-’.
-1. `finalizers`: added by COSI to defer `BucketAccess` deletion until backend deletion ops succeed.
+1. `finalizers`: added by COSI to defer BucketAccess deletion until the pod that uses this access terminates, and the related `BucketAccessRequest` is deleted.
```

Comment:
+1
nit: why remove the back-quotes on BucketAccess?

Reply:
crap, that is cut and paste error